.github/packaging/pre_build_gpu.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Builds vLLM, Monarch and torchstore
+# This script builds vLLM, Monarch and torchstore and places
+# their wheels into dist/.
+
+MONARCH_COMMIT="265034a29ec3fb35919f4a9c23c65f2f4237190d"
+TORCHTITAN_COMMIT="82f0287b966f1735819a377a9a09e7a303c55faa"
```

Comment:
nightly?

Reply:
This is actually not used. I think we really only need to successfully upload Monarch and vLLM here, the rest can be handled elsewhere